### PR TITLE
Add JOSS paper draft.

### PR DESCRIPTION
**Pull request by orionarcher**
## Description
This includes the paper and code necessary for a SolvationAnalysis JOSS submission.

### CODE REVIEW

File: joss_paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+# Figures
+
+![A summary of SolvationAnalysis capabilities. \label{fig:summary}](docs/assets/summary_figure.jpg)
+
+# Statement of need
+
+Molecular dynamics studies of liquid solvation structures often replicate
+established analyses on novel systems. In electrolyte systems, it is common
+to calculate coordination numbers, radial distribution functions, solute
+dissociation, cluster speciation, etc. In principle, these analyses are highly
```
Comment (hmacdope) [inline joss_paper/paper.md:67]: with my JOSS reviewer hat on this probably needs a citation.

File: joss_paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+Molecular dynamics studies of liquid solvation structures often replicate
+established analyses on novel systems. In electrolyte systems, it is common
+to calculate coordination numbers, radial distribution functions, solute
+dissociation, and cluster speciation [@Hou:2019]. In principle, these analyses are highly
+similar across a diversity of systems. In practice, many specialized bespoke
+tools have sprung up to address the same underlying problem. Enter `SolvationAnalysis`, 
+an easy-to-use Python package with an interactive interface for
+computing a wide variety of solvation properties. Building on `MDAnalysis` and
+`pandas` [@Michaud-Agrawal:2014] [@Gowers:2016] [@pandas:2020], it efficiently
+processes output from a wide variety of Molecular Dynamics applications.
```
Comment (hmacdope) [inline joss_paper/paper.md:72]:
```suggestion
processes output from a wide variety of molecular dynamics simulation packages.
```

File: joss_paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+`SolvationAnalysis` was designed to free researchers from laboriously
+implementing common calculations. In addition to routine properties like
```
Comment (hmacdope) [inline joss_paper/paper.md:75]:
```suggestion
implementing and validating the correctness of common analysis calculations. In addition to routine properties like
```

File: joss_paper/paper.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+SolvationAnalysis uses tools from the SciPy ecosystem [@numpy:2020] [@scipy:2020]
+to implement analyses of network formation [@Wang:2022] and residence
+times [@Self:2019], summarized in \autoref{fig:summary}. To make visualization fast, 
+the package includes a robust set of plotting tools built
+on top of `Matplotlib` and `Plotly` [@matplotlib:2007] [@plotly:2015].
+A full set of tutorials based on state-of-the-art electrolyte systems 
+[@Hou:2019] [@Dong-Joo:2022] are also included to familiarize new researchers
+with solvation structure analysis. Together, these features allow for
+rapid interactive or programmatic calculation of solvation properties.
+
```
Comment (hmacdope) [inline joss_paper/paper.md:86]: Maybe add something about enabling high throughput analysis? Just a thought

**Review by hmacdope — CHANGES_REQUESTED** [review]
Some tiny suggestions

File: joss_paper/paper.md (hunk 4 above)
Comment (orionarcher) [reply joss_paper/paper.md:85]: I had something about high throughput analysis but I decided to take it out. I'll start using that language once I have a better way to do high throughput simulations :)

**Comment by rkingsbury** [comment]
Thanks @orionarcher ! Please update my affiliation to the following:

```
- name: Energy Storage & Distributed Resources Division, Lawrence Berkeley National Laboratory, USA
```

**Review by hmacdope — APPROVED** [review]
LGTM! ready to go IMO.

**Comment by orionarcher** [comment]
Waiting to finish up some fine tuning of the plotting module in #85, then will merge! Probably in next day or two.

**Pull request merged**